js-exercises/sum-all/sumAll.js

```diff
@@ -0,0 +1,14 @@
+function sort(inputIntegerArray) {
+  return inputIntegerArray.sort((a, b) => a - b);
+}
+
+function sumAll(rangeArray) {
+  sort(rangeArray);
+  let sum = 0;
+  for (let i = rangeArray[0]; i <= rangeArray[1]; i += 1) {
```

Comment:
Try to use JavaScript Array functions like reduce, forEach or map over here. They would help reduce bugs.